packages/obojobo-express/models/media.js

```diff
@@ -0,0 +1,296 @@
+const fs = require('fs')
+const sharp = require('sharp')
+
+const mediaConfig = oboRequire('config').media
+const logger = oboRequire('logger.js')
+const db = oboRequire('db')
+
+class Media {
+	static parseCustomImageDimensions(dimensionsAsString) {
+		let width, height
+
+		const parsedDimensions = dimensionsAsString.split('x'),
```

Comment:
here also 
```
const parsedDimensions = dimensionsAsString.split('x')
const finalDimensions = {}
```

Reply:
Fixed in commit 8cec342957dee6b9491bc688969d66179d06b049